### PR TITLE
Add Russia

### PR DESCRIPTION
**Pull request by systemcatch**
This is an updated version of #766 . I've made the requested changes and got it working with Python 3. Also rebased the branch to bring it up to date. Lets get Russia on the map! :ru: 

/cc @alixunderplatz , @denplis 

ref #757 

ru

### CODE REVIEW

File: parsers/requirements.txt [hunk 1]
```diff
@@ -9,3 +9,4 @@ reescraper==1.6.0
 requests==2.18.4
 requests-mock==1.3.0
 xlrd==1.0.0
+tablib==0.12.1
```
Comment (corradio) [inline parsers/requirements.txt]: Can you order alphabetically?

**Comment by systemcatch** [comment]
I think it is measured once the the current hour is past (does that make sense?). @alixunderplatz you've done the work on this and can explain it better than me.

**Comment by corradio** [comment]
ok waiting for confirmation from @alixunderplatz before merging!

**Comment by alixunderplatz** [comment]
@corradio In #757 I assumed it was a "final schedule" for the upcoming hour.
You can track it [here](http://br.so-ups.ru/Public/MainPageData/BR/PowerGeneration.aspx) and see that the upcoming interval will change around minute :51 / :52 of each hour.

Just right now, values for "interval 15" (and following hours) were updated, while for "interval 14" they remain unchanged.

image

Assuming "interval 15" is 14-15 moscow time, it was "over" just in these minute. So either it is measured in an odd time frame (13:50 to 14:50) , or related to the previous (13-14?) hour and measured or something I don't understand. Maybe it is measured generation from 13:30 to 14:30 moscow time, which is published 20 min later. Or another timezone (Moscow time is central for rail grid, too!). or a final schedule. Assumptions.
Although, it clearly says "generation" or "Выработка".

I will look for a documentation that was somewhere on the site!

@systemcatch We could set 567 g/kWh to the "unknown" part as emission factor (my comment from Jan 5 #757 and Brunos replies) https://github.com/tmrowco/electricitymap/blob/master/config/co2eq_parameters.js


image

**Comment by alixunderplatz** [comment]
@corradio @systemcatch ah it's tricky! I just saw there is an "interval 0" in the chart on http://br.so-ups.ru/Public/MainPageData/BR/PowerGeneration.aspx

So "interval 15" probably could mean "15 to 16", for which the data is updated 10 min ahead.
Will look for the documentation!

**Comment by alixunderplatz** [comment]
Couldn't get anything out of the documentation of the publicly available data, because they left the third tab "БР" out ^^

On their main page they have the graph for "generation in the 15th hour" 

http://br.so-ups.ru/Public/MainPage.aspx
Unfortunately, clicking on "15 час" will just foward you to the generation table and not to the most recent hour.

In the top row it says "ч" which is the abbreviation for "час", so "hour".

image

I suggest interval "0" is 00:00 to 01:00. This is also supported by the time selector on the main page above the map in the upper right corner.

image

At 16:00 moscow time, data for the interval 14 to 15 shows actual generation on the map, too. So actual data is published with a delay of 1 hour. For interval 15 to 16 it is still empty (-), but planned generation is filled. But no seperation of types in that case.

I still think the data from the generation by type page is the "final 10-min-ahead schedule". 
Looking on the map on the front page, you can see that the differences between planned (План.) and actual (Факт.) are rather insignificant compared to the > 100 GW generation.

Relying on a final schedule for the upcoming hour seems legit.

**Comment by systemcatch** [comment]
Currently the upcoming hour is dropped in the code, so at 3.59pm data for 4.00pm will still be ignored. This seems safer than including it for now. Otherwise I think we are ok with the parser as currently constructed.

**Comment by alixunderplatz** [comment]
@systemcatch good!

By the way, it looks like Kaliningrad is also filled with colour. It will have no effect on exports to Lithuania, because that is defined as "from RU-KGD", right?

As a consecutive step, we should consider seperating the price zones by federal regions (called "oblast" or "krai"). Just like the states in the US. This will exclude the northern islands and thinly populated parts of Siberia. Will drop a list in a seperate issue when I have it.

**Comment by corradio** [comment]
Awesome! Let's merge this and iterate.

**Comment by brunolajoie** [comment]
💪  Be ready for some twitter activity!

**Pull request merged**